app/controllers/users_controller.rb

```diff
@@ -1,3 +1,29 @@
 class UsersController < ApplicationController
-  def new; end
+  def new
+    @user = User.new
+  end
+
+  def show
+    @user = User.find_by id: params[:id]
+    unless @user
+      flash[:danger] = t ".not_found_user"
+      redirect_to root_url
+    end
+  end
+
+  def create
+    @user = User.new user_params
+    if @user.save
+      flash[:success] = t ".welcome"
+      redirect_to root_url
+    else
+      render :new
+    end
+  end
+
+  private
+
+  def user_params
+    params.require(:user).permit User.user_st_params
```

Comment:
dùng trực tiếp nha em
```suggestion
    params.require(:user).permit User::ATTR_PARAMS
```